source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -188,31 +189,33 @@
     'MINUTES': 2880,
 }
 
+# Email service configuration (using Anymail).
+# Refer to Anymail's documentation for configuration details.
+
+ANYMAIL = {
+    "SENDINBLUE_API_KEY": "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",
```

Comment:
This line is a security leak, i've ping @francisjeanneau  to fix this with @yanicolivier.

@yanicolivier can you fix this on your server as soon as possible ? 

 - Change the API KEY on sendinblue platform
 - Change the old key in your server settings